GVFS/GVFS/CommandLine/CloneVerb.cs

```diff
@@ -455,6 +455,11 @@ private void CheckNotInsideExistingRepo(string normalizedEnlistmentRootPath)
             {
                 this.ReportErrorAndExit("Error: You can't clone inside an existing GVFS repo ({0})", existingEnlistmentRoot);
             }
+
+            if (this.IsExistingPipeListening(normalizedEnlistmentRootPath))
+            {
+                this.ReportErrorAndExit($"Error: It appears there is currently a GVFS.Mount process running for ({normalizedEnlistmentRootPath}). This process must be stopped before cloning.");

```

Comment:
Minor style suggestion:  I think you can probably drop the "It appears" from the message and simply leave it at "There is currently a GVFS.Mount ..."